Amplify/Categories/Predictions/Models/LabelType.swift

```diff
@@ -5,8 +5,10 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-public enum LabelType {
-    case all
-    case moderation
-    case labels
+extension Predictions {
+    public enum LabelType {
```

Comment:
Just checking: not replacing this for a `struct` is expected?

Reply:
😬  thanks for catching this. Addressed in [81ff28b](https://github.com/aws-amplify/amplify-swift/pull/2892/commits/81ff28b5589272e04698f19517a13f1dc663fb06)